nodejs/week1/avg.js

```diff
@@ -0,0 +1,11 @@
+const args = process.argv.slice(2);
```

Comment:
Good idea to extract this into a variable! Makes the code a lot easier to read. 💯